.gitlab-ci.yml

```diff
@@ -5,19 +5,22 @@ include:
 
 variables:
   SERVICE_PORT: 80
+  DOCKER_BUILD_ARG_REACT_APP_SENTRY_DSN: '$GL_SENTRY_DSN'
+  DOCKER_BUILD_ARG_SASS_PATH: 'node_modules:src/assets/styles'
 
 build-review:
   extends: .build
   variables:
     DOCKER_IMAGE_NAME: "$CI_PROJECT_NAME-review"
-    DOCKER_BUILD_ARG_REACT_APP_SERVICE_MAP_URI: "https://palvelukartta.hel.fi/unit/"
-    DOCKER_BUILD_ARG_REACT_APP_API_URI: https://venepaikka-federation.test.kuva.hel.ninja/
-    DOCKER_BUILD_ARG_REACT_APP_TUNNISTAMO_URI: https://tunnistamo.test.kuva.hel.ninja
+    DOCKER_BUILD_ARG_REACT_APP_API_URL: "https://venepaikka-federation.test.kuva.hel.ninja/"
+    DOCKER_BUILD_ARG_REACT_APP_SERVICE_MAP_URL: "https://palvelukartta.hel.fi/unit/"
```

Comment:
This seems to be same for all envs, so could we move it under upper `variables`?

Reply:
Done